### ISSUE TITLE
"Category Other cannot be renamed"

### ISSUE THREAD

**Issue by HighPriest** [Medvedev91/timetome#19]
It looks like you are identifying categories by names instead of unique IDs. It shouldn't matter what a category is named. If it is a default, maybe it shouldn't be removable, but not, not renamble.

**Comment by appstome** [comment]
No, I use unique IDs. There are many internal reasons I made "Other" non-renamed. Tell me please, why you need to rename Other? It's a very versatile activity.

**Comment by arthurbacci** [comment]
> No, I use unique IDs. There are many internal reasons I made "Other" non-renamed. Tell me please, why you need to rename Other? It's a very versatile activity.

It doesn't make much sense really, but the possibility of make "Other" not keep the screen on might be useful for some people I think

**Comment by Medvedev91** [comment]
Due to upcoming release #59, the issue makes sense.

**Comment by 229c9cf0** [comment]
It'd be useful to change
- name (to a different language)
- emoji (light bulb maybe matches "idea" but not "other")
- color
- disable "keep screen on"
and probably more… so "Other" being uneditable just means it's effectively a dead entry in the list that I can't / don't want to use for anything, since I can't make it conform to my expectations.

So if it stays uneditable, a better name might be "uncategorised" or similar, at which point it can simply be an implicit entry that doesn't even show up in that list.

**Comment by Medvedev91** [comment]
Fixed https://github.com/Medvedev91/timeto.me/releases/tag/v2025.06.26.580
